index.html

```diff
@@ -8,6 +8,6 @@
     <title>Document</title>
```

Comment:
[OPTIONAL] It's a great idea to also give your application a relevant title, in this case, "Hello Microverse". The title will always show on every tab of the webpage.